conformance/03_blob_upload_monolithic_test.go

```diff
@@ -18,6 +18,25 @@ var test03BlobUploadMonolithic = func() {
 			Expect(resp.StatusCode()).To(Equal(http.StatusNotFound))
 		})
 
+		g.Specify("POST request with digest and blob should yield a 201", func() {
+			req := client.NewRequest(reggie.POST, "/v2/<name>/blobs/uploads/").
+				SetHeader("Content-Length", configContentLength).
+				SetHeader("Content-Type", "application/octet-stream").
+				SetQueryParam("digest", configDigest).
+				SetBody(configContent)
+			resp, err := client.Do(req)
+			Expect(err).To(BeNil())
+			Expect(resp.StatusCode()).To(Equal(http.StatusCreated))
```

Comment:
https://github.com/opencontainers/distribution-spec/blob/master/spec.md#on-success-created-1

Needs a Location and Session-ID check (whatever this check means). I would definitely add at least the Location check.

Reply:
Thanks for the heads up.  We've added a location check anywhere there's a POST request or we are expecting a 201 (Created) response.